common4j/src/main/com/microsoft/identity/common/java/util/CommonURIBuilder.java

```diff
@@ -159,4 +158,14 @@ private boolean containsParam(@NonNull final String param){
         return false;
     }
 
+    /**
+     *
+     * @return the last segment in the URI. Returns an empty string if there are no path segments.
+     */
+    public String getLastPathSegment() {
+        final List<String> pathSegments = getPathSegments();
+        if (pathSegments.isEmpty()) return "";
```

Comment:
Please the code format that we have in the library for `if` statements

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1753/commits/f58ff6ec019f1ebe2587d702c6e465d7acbbea0f